--- tomboy-0.3.2.orig/Tomboy/panelapplet/AppletFactory.cs	23 Oct 2004 18:21:58 -0000	1.1
+++ tomboy-0.3.2/Tomboy/panelapplet/AppletFactory.cs	8 Jun 2005 10:31:47 -0000	1.2
@@ -4,10 +4,10 @@
 
 namespace PanelApplet
 {
-	public class AppletFactory
+	public class AppletFactory : GLib.Object
 	{
-		private AppletFactory () { }
+		private AppletFactory () : base (IntPtr.Zero) { }
 
 		private static string _IID;
 		private static string _factoryIID;
