Description: Use gmime 2.6 instead of 2.4
Author: Michael Biebl <biebl@debian.org>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=663216

Index: tomboy/configure.in
===================================================================
--- tomboy.orig/configure.in	2012-06-05 14:28:11.806991429 +0100
+++ tomboy/configure.in	2012-06-05 14:31:50.962981784 +0100
@@ -235,8 +235,8 @@
 # Evolution.dll needs GMime#
 #
 if test "x$ENABLE_GNOME" != "xno"; then
-	if pkg-config --exists gmime-sharp-2.4; then
-		PKG_CHECK_MODULES(EVOLUTION, gmime-sharp-2.4, ENABLE_EVOLUTION="yes", ENABLE_EVOLUTION="no")
+	if pkg-config --exists gmime-sharp-2.6; then
+		PKG_CHECK_MODULES(EVOLUTION, gmime-sharp-2.6, ENABLE_EVOLUTION="yes", ENABLE_EVOLUTION="no")
 	else
 		PKG_CHECK_MODULES(EVOLUTION, gmime-sharp >= 2.2.7, ENABLE_EVOLUTION="yes", ENABLE_EVOLUTION="no")
 	fi
